### ISSUE TITLE
Containers/Projects table sorting failure in 'Display Name' column

### ISSUE THREAD

**Issue by gshefer** [ManageIQ/integration_tests#4052]
On Compute->Containers->Projects:
        sort_tbl.sort_by('Display Name', 'ascending')

Exception:
        Detected malfunction in table ordering (wanted Display Name, got Name)

Example:
    Test and line:  https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/containers/test_containers_main_pages_sort.py#L42

Traceback: 

cls = <class 'cfme.containers.project.Project'>

    @pytest.mark.meta(blockers=[
        BZ(1392413, unblock=lambda cls: cls != ContainersProvider),
        BZ(1409360, unblock=lambda cls: cls != ContainersProvider)
    ])
    @pytest.mark.parametrize('cls', TEST_OBJECTS)
    def test_containers_main_pages_sort(cls):
    
        navigate_to(cls, 'All')
        toolbar.select('List View')
        # NOTE: We must re-instantiate here sort_tbl in order to prevent StaleElementException
        sort_tbl = SortTable(table_locator="//div[@id='list_grid']//table")
        header_texts = [header.text for header in sort_tbl.headers]
        for col, header_text in enumerate(header_texts):
    
            if not header_text:
                continue
    
            # Checking both orders
>           sort_tbl.sort_by(header_text, 'ascending')

cfme/tests/containers/test_containers_main_pages_sort.py:46: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <cfme.web_ui.SortTable _loc="//div[@id='list_grid']//table">
header = 'Display Name', order = 'ascending'

    def sort_by(self, header, order):
        """Sorts the table by given conditions
    
            Args:
                header: Text of the header cell to use for sorting.
                order: ascending or descending
            """
        order = order.lower().strip()
        if header != self.sorted_by:
            # Change column to order by
            self.click_header_cell(header)
            if self.sorted_by != header:
                raise Exception(
                    "Detected malfunction in table ordering (wanted {}, got {})".format(
>                       header, self.sorted_by))
E               Exception: Detected malfunction in table ordering (wanted Display Name, got Name)

cfme/web_ui/__init__.py:946: Exception

![image](https://cloud.githubusercontent.com/assets/21076742/22216612/771e107e-e1a8-11e6-9ddf-f542a62151f7.png)

**Comment by mfalesni** [comment]
@gshefer  Does it work manually? Seems like it gets clicked, but the name column remains the source of sorting.

**Comment by gshefer** [comment]
@mfalesni yes, it's working, but the problem is the "Display Name" column.

**Comment by mfalesni** [comment]
@gshefer Please, send me the IP address (privately) of an appliance with openshift set up and I can take a look.

**Comment by mfalesni** [comment]
I have just been playing with it and I cannot reproduce this issue when trying it out of ipython.

**Comment by dajohnso** [comment]
@mfalesni what is the latest here, did @gshefer send you what you need, any estimate as to when we might be able to knock this out?  I mean its a table sorting issue so not a critical test but we should fix it and it would seem fairly straight-forward.

**Comment by mfalesni** [comment]
@dajohnso I responded on the e-mail related to this issue. Possible Selenium woes maybe.

**Comment by gshefer** [comment]
Issue solved for some reason. Closing...
